src/app/static/src/tests/e2e/mint.etest.ts

```diff
@@ -8,6 +8,17 @@ test.describe("basic tests", () => {
         expect(await page.innerText("a.navbar-brand")).toBe("MINT");
     });
 
+    test("default prevalence is 26-35%", async ({page}) => {
+        await page.goto("/");
+
+        await page.fill("#name", "Project 1");
+
+        await page.fill(".ti-new-tag-input", "Region A");
+        await page.click(".btn-primary");
+
+        await expect(page.locator("select[name=currentPrevalence] option:checked")).toHaveText(/^\s*26-35%\s*$/);
```

Comment:
`toHaveText("26-35%")` isn't suitable due to surrounding whitespace

`toContainText("26-35%")` is too lax - the following would pass as well, but isn't useful: `expect(page.locator("select[name=currentPrevalence]")).toContainText("26-35%")`